2024-05-15 16:38:39,203 INFO    MainThread:95278 [wandb_setup.py:_flush():76] Current SDK version is 0.17.0
2024-05-15 16:38:39,203 INFO    MainThread:95278 [wandb_setup.py:_flush():76] Configure stats pid to 95278
2024-05-15 16:38:39,203 INFO    MainThread:95278 [wandb_setup.py:_flush():76] Loading settings from /Users/maheepchaudhary/.config/wandb/settings
2024-05-15 16:38:39,203 INFO    MainThread:95278 [wandb_setup.py:_flush():76] Loading settings from /Users/maheepchaudhary/pytorch/Projects/concept_eraser_research/DAS_MAT/wandb/settings
2024-05-15 16:38:39,203 INFO    MainThread:95278 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-05-15 16:38:39,203 INFO    MainThread:95278 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-05-15 16:38:39,203 INFO    MainThread:95278 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-05-15 16:38:39,203 INFO    MainThread:95278 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-05-15 16:38:39,203 INFO    MainThread:95278 [wandb_init.py:_log_setup():520] Logging user logs to /Users/maheepchaudhary/pytorch/Projects/concept_eraser_research/DAS_MAT/wandb/run-20240515_163839-fj3ft764/logs/debug.log
2024-05-15 16:38:39,203 INFO    MainThread:95278 [wandb_init.py:_log_setup():521] Logging internal logs to /Users/maheepchaudhary/pytorch/Projects/concept_eraser_research/DAS_MAT/wandb/run-20240515_163839-fj3ft764/logs/debug-internal.log
2024-05-15 16:38:39,203 INFO    MainThread:95278 [wandb_init.py:_jupyter_setup():466] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x42c0efcb0>
2024-05-15 16:38:39,203 INFO    MainThread:95278 [wandb_init.py:init():560] calling init triggers
2024-05-15 16:38:39,203 INFO    MainThread:95278 [wandb_init.py:init():567] wandb.init called with sweep_config: {}
config: {}
2024-05-15 16:38:39,203 INFO    MainThread:95278 [wandb_init.py:init():610] starting backend
2024-05-15 16:38:39,203 INFO    MainThread:95278 [wandb_init.py:init():614] setting up manager
2024-05-15 16:38:39,204 INFO    MainThread:95278 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2024-05-15 16:38:39,204 INFO    MainThread:95278 [wandb_init.py:init():622] backend started and connected
2024-05-15 16:38:39,205 INFO    MainThread:95278 [wandb_run.py:_label_probe_notebook():1328] probe notebook
2024-05-15 16:38:39,205 INFO    MainThread:95278 [wandb_run.py:_label_probe_notebook():1338] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-05-15 16:38:39,205 INFO    MainThread:95278 [wandb_init.py:init():711] updated telemetry
2024-05-15 16:38:39,220 INFO    MainThread:95278 [wandb_init.py:init():744] communicating run to backend with 90.0 second timeout
2024-05-15 16:40:09,564 ERROR   MainThread:95278 [wandb_init.py:init():770] encountered error: Run initialization has timed out after 90.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
2024-05-15 16:40:09,565 ERROR   MainThread:95278 [wandb_init.py:init():1174] error in wandb.init()
Traceback (most recent call last):
  File "/Users/maheepchaudhary/miniforge3/envs/nnsight/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 1164, in init
    return wi.init()
           ^^^^^^^^^
  File "/Users/maheepchaudhary/miniforge3/envs/nnsight/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 776, in init
    raise error
wandb.errors.CommError: Run initialization has timed out after 90.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
